js/app/Visualization/UI/Widgets/Timeline/Timeline.less

```diff
@@ -196,7 +208,7 @@
         width: 100%;
 
         white-space: nowrap;
-        font-size: 0;
+        font-size: 10pt;
```

Comment:
@jarmstrongbts This looks a bit dangerous. font-size: 0 is generally a trick for box sizing. Why did you change this?